### PR TITLE
Filter teams based on user ou

### PR DESCRIPTION
**Pull request by weradsaoud**

### CODE REVIEW

**Review by aalshahin — CHANGES_REQUESTED** [review]
this changes is needed for approving

File: src/modules/administration/popups/internal-user-ou-popup/internal-user-ou-popup.component.ts [hunk 1]
```diff
@@ -52,14 +54,23 @@ export class InternalUserOUPopupComponent extends AdminDialogComponent<InternalU
       )
       .subscribe(filteredData => {
         this.organizationUnits = filteredData;
+        console.log('this.organizationUnits: ', this.organizationUnits);
```
[the remark this answers is not in the document]
Comment (weradsaoud) [reply src/modules/administration/popups/internal-user-ou-popup/internal-user-ou-popup.component.ts:57]: done

File: src/modules/administration/popups/internal-user-ou-popup/internal-user-ou-popup.component.ts [hunk 2]
```diff
@@ -52,14 +54,23 @@ export class InternalUserOUPopupComponent extends AdminDialogComponent<InternalU
...
+
+        this.administrationAndSectionUnits = this.organizationUnits.filter(
+          o =>
+            o.type === OrganizationUnitType.ADMINISTRATION ||
+            o.type === OrganizationUnitType.ADMINISTRATION,
```
[the remark this answers is not in the document]
Comment (weradsaoud) [reply src/modules/administration/popups/internal-user-ou-popup/internal-user-ou-popup.component.ts:62]: done

File: src/modules/administration/popups/organization-unit-popup/organization-unit-popup.component.ts [hunk 3]
```diff
@@ -108,7 +108,11 @@ export class OrganizationUnitPopupComponent extends AdminDialogComponent<Organiz
     if (ouId)
       this.internalUserService
         .getInternalUsersByOuId(ouId)
-        .subscribe(internalUsers => (this.managerAssistants = internalUsers));
+        .subscribe(internalUsers => {
+          console.log('internalUsers', internalUsers);
```
[the remark this answers is not in the document]
Comment (weradsaoud) [reply src/modules/administration/popups/organization-unit-popup/organization-unit-popup.component.ts:112]: done

File: src/modules/administration/popups/internal-user-ou-popup/internal-user-ou-popup.component.html [hunk 4]
```diff
@@ -12,10 +12,21 @@ <h1>{{ lang.map.menu_internal_user_ou }}</h1>
...
             bindLabel="getNames"
             [label]="lang.map.organization_unit"
+            formControlName="administrationAndSectionUnit"
+            [isMultiple]="false"
+            [enableFilter]="true"
+            bindFilter="getNames"></app-select-input>
+        </div>
+        <div class="mt-4">
+          <app-select-input
+            [options]="departmentUnits"
```
[the remark this answers is not in the document]
Comment (weradsaoud) [reply src/modules/administration/popups/internal-user-ou-popup/internal-user-ou-popup.component.html:26]: done

**Pull request closed without merging**